File: easy/Swap-Elements/Swap Elements.cs
using System;
using System.IO;
using System.Collections.Generic;

class Program
{
    static void Main(string[] args)
    {
        using (StreamReader reader = File.OpenText(args[0]))
        while (!reader.EndOfStream)
        {
            string line = reader.ReadLine();
            if (null == line)
                continue;
            // do something with line
            ShoNums(line);
        }
    }
    
    public static void ShoNums(string line){
        string result = "";
        int pos  = line.IndexOf(":");
        string[] nums = line.Substring(0,pos).Split(' ');
        string[] swaps = line.Substring(pos+1).Split(',');
        for(int i=0;i<swaps.Length;i++){
            string sw = swaps[i].Trim();
            int dashpos = sw.IndexOf('-');
            int num1 = Convert.ToInt32(sw.Substring(0,dashpos));
            int num2 = Convert.ToInt32(sw.Substring(dashpos+1));
            string temp = nums[num1];
            nums[num1] = nums[num2];
            nums[num2] = temp;
        }
        for(int i=0;i<nums.Length;i++)Console.Write(nums[i] + " ");
        Console.WriteLine();
    }
}

File: easy/Swap-Elements/Swap-Elements.java
import java.io.*;
import java.util.*;
public class Main {
    public static void main (String[] args) throws IOException {
        File file = new File(args[0]);
        BufferedReader buffer = new BufferedReader(new FileReader(file));
        String line;
        while ((line = buffer.readLine()) != null) {
            line = line.trim();
            // Process line of input Here
            ShoNums(line);
        }
    }
    
    public static void ShoNums(String line){
        String result = "";
        int pos  = line.indexOf(":");
        String[] nums = line.substring(0,pos).split(" ");
        String[] swaps = line.substring(pos+1).split(",");
        for(int i=0;i<swaps.length;i++){
            String sw = swaps[i].trim();
            int dashpos = sw.indexOf("-");
            int num1 = Integer.valueOf(sw.substring(0,dashpos));
            int num2 = Integer.valueOf(sw.substring(dashpos+1));
            String temp = nums[num1];
            nums[num1] = nums[num2];
            nums[num2] = temp;
        }
        for(int i=0;i<nums.length;i++)System.out.print(nums[i] + " ");
        System.out.println();
    }
}
